.npmignore

```diff
@@ -0,0 +1,2 @@
+example01/
```

Comment:
Could you perhaps change this to `example*` to account for the new example that will be added in #10?

Reply:
Good call. I just saw the PR for `example02` so this change makes sense - I just pushed it.